lib/broccoli/default-packager.js

```diff
@@ -562,7 +562,7 @@ module.exports = class DefaultPackager {
       for (let outputFile in this.styleOutputFiles) {
         let isMainVendorFile = outputFile === this.distPaths.vendorCssFile;
         let headerFiles = this.styleOutputFiles[outputFile];
-        let inputFiles = isMainVendorFile ? ['addon-tree-output/**/*.css'] : [];
+        let inputFiles = isMainVendorFile ? ['addon-tree-output/**/__COMPILED_STYLES__/**/*.css'] : [];
```

Comment:
This glob is needed to catch addons that have a slash in their name. E.g. if addon name is "my-org/my-addon", its compiled CSS will be at the path: `addon-tree-output/my-org/my-addon/__COMPILED_STYLES__/my-org/my-addon.css`